### ISSUE TITLE
[BUG] - Drop down trigger does not open on browser mobile inspector

### ISSUE THREAD

**Issue by habibKadiri** [nextui-org/nextui#969]
### Describe the bug

When I click on any of the drop down triggers in the docs while in the mobile inspector on chrome/firefox. The drop down does not open, I noticed this in development and wanted to test on the docs and ecnountered the same issue. I don't know if this also means that it affects real mobile devices.

### Your Example Website or App

https://nextui.org/docs/components/dropdown#static

### Steps to Reproduce the Bug or Issue

1. go to https://nextui.org/docs/components/dropdown#static
2. Open browser Inspector
3. Click on toggle device toolbar
![image](https://user-images.githubusercontent.com/60525087/209483201-888896c6-15e6-47fb-9b1d-949eace827c0.png)
4. Click on trigger in page:
![image](https://user-images.githubusercontent.com/60525087/209483214-079b0340-4c7a-428d-8c42-968baeec5de8.png)

### Expected behavior

Drop down should open

### Screenshots or Videos

[screen-capture (28).webm](https://user-images.githubusercontent.com/60525087/209483225-2d22a3a5-5ed3-47aa-8944-cd5049d01794.webm)

### Operating System Version

Linux

### Browser

Chrome

**Comment by joaoGabriel55** [comment]
I can take a look it

**Comment by depyronick** [comment]
I can confirm that dropdowns are not opening on the production, mobile safari browser.

**Comment by joaoGabriel55** [comment]
Testing here, the Dropdown Popover to be show in both contexts: web and mobile must follow this structure:
```tsx
<Dropdown>
    <Dropdown.Trigger> // <-- must have Trigger
        <Dropdown.Button light color="primary"> // <-- must be Dropdown.Button
             Light
        </Dropdown.Button>
    </Dropdown.Trigger>
 // ....
```

**Comment by depyronick** [comment]
```
Uncaught TypeError: Cannot read properties of undefined (reading 'contains')
    at onClick (module.js?6cc0:248:1)
    at eval (module.js?687d:171:1)
    at eval (module.js?4473:79:1)
    at onPress (button.js?7546:1:1273)
    at triggerPressEnd (module.js?6cc0:189:1)
    at HTMLDocument.onPointerUp (module.js?6cc0:345:1)
```

My code was missing `Dropdown.Trigger`, but after adding it, I get an error.

**Comment by joaoGabriel55** [comment]
Yeah, I saw this error too, after added the `Dropdown.Trigger`. I need to take another better look on code.

**Comment by joaoGabriel55** [comment]
Update: I tested on Firefox and works the trigger event on mobile device mode

**Comment by joaoGabriel55** [comment]
> Yeah, I saw this error too, after added the `Dropdown.Trigger`. I need to take another better look on code.

Apparently on browser mobile inspector the `PopoverTrigger` that is used by `DropdownTrigger` does not update your context state. So strange, cause on Firefox browser works well.

**Comment by joaoGabriel55** [comment]
This bug was already solved on this PR: https://github.com/nextui-org/nextui/pull/965 by @tianenpang 
I tested this solution, and works perfect.

**Comment by jackcusick95** [comment]
hey, wondering when your team plans to merge the mobile trigger fix into prod?

**Comment by cini9** [comment]
Any updates on this ? This really is a bit of a blocking issue.

I've also tested with extendVariants like this :

```
/**
 * The external imports
 */
import { Button as HeroUIButton } from '@heroui/button'
import { extendVariants } from '@heroui/system'

const Button = extendVariants(HeroUIButton, {
  variants: {
    variant: {
      ghost: 'text-default',
      bordered: 'text-default',
      light: 'data-[hover=true]:bg-foreground',
    },
    size: {
      sm: 'text-sm',
    },
  },
  defaultVariants: {
    radius: 'full',
    disableRipple: 'true',
    size: 'md',
    color: 'primary',
  },
})

export default Button

```
And the dropdowns don't seem to work on mobile.
